Tools/scripts/summarize_stats.py

```diff
@@ -350,35 +360,38 @@ def emit_execution_counts(opcode_stats, total):
             rows
         )
 
+def _emit_comparative_execution_counts(base_rows, head_rows):
+    base_data = dict((x[0], x[1:]) for x in base_rows)
+    head_data = dict((x[0], x[1:]) for x in head_rows)
+    opcodes = set(base_data.keys()) | set(head_data.keys())
```

Comment:
Dictionary keys support set operations:

```suggestion
    opcodes = base_data.keys() | head_data.keys()
```

Reply:
TIL!